sonarts-sq-plugin/sonar-typescript-plugin/src/test/java/org/sonar/plugin/typescript/ExternalTypescriptSensorTest.java

```diff
@@ -383,4 +381,24 @@ protected void readErrors(BufferedReader errorReader) {
     }
   }
 
+  private static class SlowErrorConsumer extends ExternalProcessErrorConsumer {
```

Comment:
@saberduck I would change the name of this one to `TestableErrorConsumer`, make the duration of sleep a constructor parameter and set it to `0` for all test cases but one.